parent/pom.xml

```diff
@@ -241,11 +241,12 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
```

Comment:
the whole intent is to not *require* running the testsuites and just be able to add Test classes without the Testsuite.

if you enable this as things are now you get the worst part of both worlds IMO. 

The old behavior was *not* to execute everything with the word *Test* in them.

The default behavior as you will note explicitly avoids running AllTests. Your patch here enables *everything* to run - multiple times at worst.